.github/workflows/build.yml

```diff
@@ -102,23 +105,15 @@ jobs:
           # try to install them anyways, so we save an apt-roundtrip.
           sudo aptitude -o Acquire::Retries=30 install gcc-multilib g++-multilib pkg-config libgmp-dev libgmp-dev:i386 libx11-dev:i386 -y
 
-      - name: Checkout tree
-        uses: actions/checkout@v4
-
       - name: Set-up Node.js
         uses: actions/setup-node@v4
         with:
-          node-version: lts/*
+          node-version: latest
 
       - name: Set-up OCaml ${{ matrix.ocaml-compiler }}
         uses: ocaml/setup-ocaml@v3
         with:
           ocaml-compiler: ${{ matrix.ocaml-compiler }}
-          dune-cache: true
-          opam-pin: false
-
-      - run: opam install conf-pkg-config
```

Comment:
I think this is the second time you try to remove that line. It was here on purpose to work-around an opam packaging issue. Are you sure it is no longer needed

Reply:
Yes, it's no longer needed.

Reply:
To be more clear, zarith now explicitly requires it, and conf-pkg-config is work properly in the Cygwin environment.

Reply:
I don't remember this being related to zarith

Reply:
At least for now, `zarith` is the only library which requires `conf-pkg-config` (on Windows)

Reply:
I though the issue was with mingw-64-shims and its optional dep on conf-pkg-config. Here is an extract of a private discussion with @dra27 

> Oh, I see what's happening here - it's a race
> There's an instant during the recompile where the old mingw-64-shims package has been removed (so there's no x86_64-w64-mingw32-gcc shim installed) and those packages then try to call the C compiler.

@dra27 do you remember this issue, can you confirm it was fixed ?

Reply:
If I remember correctly, it was fixed in `mingw-w64-shims.0.2.0`.

Reply:
No, I have failing logs involving mingw-w64-shims.0.2.0

Reply:
https://github.com/ocsigen/js_of_ocaml/actions/runs/12629900105/job/35188670458?pr=1785

Reply:
reverted in https://github.com/ocsigen/js_of_ocaml/pull/1786